### PR TITLE
entering chapter 29

### PR DESCRIPTION
**Pull request by dcpaschall**

### CODE REVIEW

**Comment by jonasmalacofilho** [comment]
@aszasz and @dcpaschall: can I already merge this?

**Comment by aszasz** [comment]
There is a table to fix and an equation to enter: with that the chapter will be semantically fine... BUT I don't know how to proceed after that because almost all the sections of this chapter have an issue I have seen in a few sections of other chapters:
- There is much more images than text, and in html we end up with a lot of images on the right side going way down without text on the center-left. In other chapters, I balanced (even sub-sections) by setting the nicest pictures with `\figure[MEDIUM]`... but that is obviously a workaround on a case-by-case basis...

I am not sure whether I should fix this case by case (so it will be nice for now, but it spoils the idea of letting the generator doing his job), or if we should have this as a  bigger issue... I would suggest I do both (fix for now and open an issue as an improvement).

Besides that, there are this other things I noticed:
- Some images that show on the html (29.54 and 29.55 on section 29.4) appear as a black square on the pdf
- html is showing tables with only two columns (29.1 and 29.4 sections 1 and 4) with full width (They are weird on pdf too, but I understand that that is 'under construction').

File: guide/pedestrian-access/pedestrian-access.src [hunk 1]
```diff
@@ -595,7 +582,8 @@ It is fairly typical in lower-income countries for distances between intersectio
...
 
 
-Chapter 29 References:
+\highlight{Chapter 29 References:}
```
Comment (jonasmalacofilho) [inline guide/pedestrian-access/pedestrian-access.src:585]: Wouldn't a proper `\section` for the references be better?

**Comment by jonasmalacofilho** [comment]
Things to do here still:

 - [x] finish adding figure size hints (~~@aszasz~~ @jonasmalacofilho)
 - ~~[x] table to fix (@aszasz)~~
 - [x] equation to enter (~~@aszasz~~ @jonasmalacofilho)
 - [x] check what's going on with some images on the pdf (@jonasmalacofilho)
 - [x] check some issues on _HTML_ tables (@jonasmalacofilho)
 - [x] (maybe) change the reference heading from `\highlight` to `\section` (~~@aszasz~~ @jonasmalacofilho)

**Comment by aszasz** [comment]
@aegauthier, would you be kind to provide the names of authors of Chapter 29.

**Comment by aszasz** [comment]
@jonasmalacofilho, Figures 29.54 and 29.55 loaded in  the pdf on-line (it takes sometime to reapear in the browser if you go away and comeback... but they are there). My local copy and TeX log are long gone... so you can not really check it. : )

**Comment by aegauthier** [comment]
Authors for 29
Contributors:
Chris Kost, ITDP Africa; Shreya Gadepalli, ITDP India; Michael King, BuroHappold Engineering

**Comment by jonasmalacofilho** [comment]
- the issue with the images on the pdf was indeed their huge (and unnecessary, given the actual content resolution) size
 - the HTML tables were working fine; it's just that they had been manually set to be large

I don't have push access to @dcpaschall, so I'll merge this as is and push my fixes directly on to the resulting master.

**Comment by jonasmalacofilho** [comment]
I batch converted all images to what seemed a high resolution but sensible size (and quality setting) using ImageMagik's `convert`:

```
for f in $(ls -S *.jpg); do echo $f $(identify -format '%Q' $f); convert $f -resize 8294400@\> -quality 90 $f; done
```

This means:

 - reduce images to have a pixel area the same as a 4K monitor (8.3M pixels)
 - set the JPEG quality to 90, which is still high enough

**Pull request merged**